### ISSUE TITLE
JSONRendered to use app's json_encoder

### ISSUE THREAD

**Issue by lalstef** [flask-api/flask-api#68]
It would be good if one could change the default json_encoder class used for serialization. The one which is currently used is `flask.json.JSONEncoder`.  Trying to change it like so 
```
app = FlaskAPI('the_app')
app.json_encoder = CustomJSONEncoder
```

does not have any effect.

https://github.com/tomchristie/flask-api/blob/1.0/flask_api/renderers.py#L15

Wouldn't something like
```
        return json.dumps(data, cls=current_app.json_encoder, ensure_ascii=False)
```

instead of 

```
        return json.dumps(data, cls=JSONEncoder, ensure_ascii=False)
```

solve the problem?

**Comment by jacebrowning** [comment]
This seems useful to me. Would you be interested in working on a PR?

**Comment by lalstef** [comment]
Yes, sure. Only I'm not sure to which branch should I create the PR. Could you please advise me on this?

**Comment by jacebrowning** [comment]
Create a PR from your fork's branch into `master`.

**Comment by jacebrowning** [comment]
Closed by https://github.com/flask-api/flask-api/pull/84

**Comment by jacebrowning** [comment]
This change is available in `Flask-API==1.1b1`.
